templates/auth/login.html

```diff
@@ -3,7 +3,7 @@
 
 {% block body %}
 <h1>{{ _('Login') }}</h1>
-<form action="{{ url_for_security('login') }}" method="POST" name="login_user_form">
+<form action="{{ url_for_security('login') }}" method="POST" name="login_user_form" class=authentication-form>
```

Comment:
If you're using this class, you may want to adapt your CSS accordingly

Reply:
yep